TrackingTools/PatternTools/src/TempTrajectory.cc

```diff
@@ -118,6 +118,8 @@ bool TempTrajectory::badForCCC(const TrajectoryMeasurement &tm) {
     return false;
   if (thit->isPixel())
     return false;
+  if (!tm.updatedState().isValid())
```

Comment:
Ciao Vincenzo,
I agree, but at this level I cannot abort PR. I can return true instead of false, but this will not guarantee to stop the PR.